client/src/components/Forum/PostCard.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles(theme => ({
+    root: {
+        width: '100%',
+        marginTop: theme.spacing(2),
+    },
+    actionArea: {
+        height: 120,
+        display: 'grid',
+        gridTemplateColumns: '120px 1fr',
+        alignItems: 'center',
+    },
+    cardInfo: {
+        display: 'grid',
+        gridTemplateRows: '1fr',
+        justify: 'space-between',
+
+        padding: 0,
+    },
+    cardImage: {
+        width: 100,
+        height: 100,
+        justifySelf: 'center',
+        borderRadius: '10px',
+    },
+    cardTitle: {
+        fontWeight: 'bold',
+        fontSize: 22,
+    },
+    cardDesc: {
+        fontSize: 14,
+    },
+    postedBy: {
+        color: '#2968FF',
+    },
+    postDateInfo: {
+        paddingTop: 10,
+    },
+}));
+
+const PostCard = props => {
+    const classes = useStyles();
+
+    return (
+        <Card className={classes.root}>
```

Comment:
I think this component needs a little more room for long text. In the image below the "Qualification" on the Google analytics post is right at the border. Maybe just adding a paddingRight would work. 

![Screen Shot 2021-01-28 at 9 10 46 AM](https://user-images.githubusercontent.com/45152761/106150663-8bbf2980-6149-11eb-9562-b24f7074210e.png)

Also I was wondering how you feel about making this a Paper component with an elevation. I think it looks more like the mockup. And I think there's a way to customize the shadow of the paper to give it that light blueish look. What is looks like using Paper with elevation=5:

![Screen Shot 2021-01-28 at 9 13 17 AM](https://user-images.githubusercontent.com/45152761/106151082-fd977300-6149-11eb-84cb-5c31cbb0fd1e.png)

Reply:
The branch I am checking has the right padding and the cards look great. 
If you try using 'rem' for font sizes let me know if you are getting any errors.